administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -14,15 +14,8 @@
 JHtml::_('formbehavior.chosen', 'select');
 
 $this->app->getDocument()->addScriptDeclaration("
-	function iframeRef( frameRef ) {
-	return frameRef.contentWindow
-		? frameRef.contentWindow.document
-		: frameRef.contentDocument
-	}
-
 	function triggerSave() {
-		var inside = iframeRef( document.getElementById('target-association') );
-		inside.getElementById('applyBtn').click();
+		window.frames[1].Joomla.submitbutton('" . $this->associatedView . ".apply');
```

Comment:
we never now if custom 3º party admin modules add more frames to the window.
the more save option is to get the iframe object by unique id